make.bat

```diff
@@ -22,6 +22,7 @@ if not defined SPHINXLINT (
 )
 
 set BUILDDIR=_build
+set SPHINXOPTS="-W --keep-going -n"
```

Comment:
```suggestion
set SPHINXOPTS=-W --keep-going -n
```

This needs to be unquoted or it doesn't work.

Reply:
And here I was assuming that without quotes it wouldn't work :upside_down_face: